<launch>
    <param name="use_sim_time" value="true"/>
    <arg name="setting_file" default="$(find semantic_slam)/setting/d435.yaml"/>
    <node name="test_semantic_slam" pkg="semantic_slam" type="test_semantic_slam" output="screen">
        <param name="vocabulary_file" value="$(find semantic_slam)/model/ORBvoc.txt"/>
        <param name="setting_file" value="$(arg setting_file)"/>

        <param name="text_graph" value="$(find semantic_slam)/model/mask_rcnn/mask_rcnn_inception_v2_coco_2018_01_28.pbtxt"/>
        <param name="model_weights" value="$(find semantic_slam)/model/mask_rcnn/frozen_inference_graph.pb"/>
        <param name="class_file" value="$(find semantic_slam)/model/mask_rcnn/object_detection_classes_coco.txt"/>
        <param name="color_file" value="$(find semantic_slam)/model/mask_rcnn/colors.txt"/>
        
        <param name="rgb_topic" value="/camera/color/image_raw"/>
        <param name="depth_topic" value="/camera/aligned_depth_to_color/image_raw"/>
    </node>
    <!-- <node name="rviz_slam" pkg="rviz" type="rviz" args="-d $(find orb_semantic_slam)/rviz/config.rviz" launch-prefix="xterm -e gdb args"/> -->
</launch>

<!-- launch-prefix="xterm -e gdb __args"-->